float64[16] th_values
float64[16] ff_values
float64[16] mf_values
float64[16] rf_values
float64[16] lf_values
float64[5] applied_force
float64[5] total_pressure
float64[5] force_deviation